tools/ui-components/src/form-control/form-control.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { FormControl, FormControlProps, FormControlVariationProps } from '.';
+
+const story = {
+  title: 'Example/FormControl',
+  component: FormControl
+};
+
+const DefaultTemplate: Story<FormControlProps> = args => {
+  return <FormControl {...args} />;
+};
+
+export const Default = DefaultTemplate.bind({});
+Default.args = {
+  // default props go here
+};
+
+const StaticTemplate: Story<FormControlVariationProps> = args => {
+  return <FormControl.Static {...args} />;
+};
+
+export const Static = StaticTemplate.bind({});
+Static.args = {
+  children: 'foo@bar.com'
+};
+
+const FeedBackTemplate: Story<FormControlVariationProps> = args => {
+  return <FormControl.Feedback {...args} />;
+};
+
+const checkMark = (
```

Comment:
This Svg is the same used in the font-awesome; however since font awesome apply styles the presentation is deferent from what we have on our feedback component.